tilings/strategy_pack.py

```diff
@@ -276,9 +276,18 @@ def insertion_row_and_col_placements(
 
     @classmethod
     def only_root_placements(
-        cls, length: int = 3, max_num_req: Optional[int] = 1,
+        cls,
+        length: int = 3,
+        max_num_req: Optional[int] = 1,
+        max_placement_rules_per_req: Optional[int] = None,
     ) -> "TileScopePack":
-        name = "only_length_{}_root_placements".format(length)
+        if max_num_req is not None:
+            name = "only_length_{length}_{max_num_req}_reqs_root_placements"
+        else:
+            name = "only_length_{length}_root_placements"
+        placement_factory = strat.RequirementPlacementFactory(
+            max_rules_per_req=max_placement_rules_per_req
+        )
```

Comment:
now I'm being picky, but shouldn't placement factory variable name just be skippd and it added to expansion strats directly- i don't really care, but it is the way we do it elsewhere

Reply:
It was ugly when I put that in the strat directly, so I moved it. Not that I really care either.

Reply:
ok, don't worry about it, lets fix the other thing and we're good to go!